README.md

```diff
@@ -14,14 +14,33 @@ npm install --save-dev @cypress/watch-preprocessor
 
 ## Usage
 
+### Prior to Cypress 10.0
+
 In your project's [plugins file](https://on.cypress.io/guides/guides/plugins.html):
 
 ```javascript
-const watch = require('@cypress/watch-preprocessor')
+const watch = require("@cypress/watch-preprocessor");
 
 module.exports = (on) => {
-  on('file:preprocessor', watch())
-}
+  on("file:preprocessor", watch());
+};
+```
+
+### Cypress 10.0 and Beyond
+
+In your project's [cypress config](https://on.cypress.io/plugins-api):
```

Comment:
Same thing as the previous PR and the link. Not sure if this link is correct or not?

Reply:
Great question: I believe so because that on link redirects to: `api/plugins/writing-a-plugin#Plugins-API` which is incorrect for `10.0` but that same slug for the unification deployment is: `https://deploy-preview-4186--cypress-docs.netlify.app/api/plugins/writing-a-plugin#Plugins-API`

Reply:
hmm. I would think we would have a special area in our docs for the new config specifically. Not sure if that exists yet or not, or if we plan on doing that. I think we probably should though. 

I mean we can always update these things later, so if you think this link is correct for now, I am happy to approve as everything else looks good to me.